assignment.rb

```diff
@@ -0,0 +1,85 @@
+class Yarn
+  attr_accessor :weight, :num_of_plies, :is_machine_wash, :yardage, :quantity
+
+  def total_yds
+    quantity * yardage
+  end
+
+  def use(type = "scarf")
+    puts "I will use this yarn to make a #{type}."
+  end
+
+  def rating
+    puts "Yarn is awesome!"
+  end
+end
+
+class Animal_Fiber < Yarn
+  attr_accessor :is_organic, :is_next_to_skin
+end
+
+class Plant_Fiber < Yarn
+  attr_accessor :is_organic
```

Comment:
If this is a boolean that can be true or false, would you reckon that I would want to ask the same question to `ManmadeFiber`? If so, I would think that this attribute could exist on the superclass